ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/SketchOnMap.qml

```diff
@@ -0,0 +1,374 @@
+        id: view
+        anchors.fill: parent
+    }
+
+    Control {
+        id: control
+        width: 110
+        anchors.right: parent.right
+        padding: 5
+
+        background: Rectangle {
+            color: "black"
+            opacity: .5
+        }
+
+        contentItem: ColumnLayout {
+            id: columns
+            anchors.verticalCenter: parent.verticalCenter
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 20
+
+            GridLayout {
+                id: geometryColumn
+                Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                columns: 2
+
+                Text {
+                    id: geometryText
+                    color: "#ffffff"
+                    text: qsTr("Geometry")
+                    font.pixelSize: 16
+                    Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+                    font.bold: true
+                    Layout.columnSpan: 2
+                }
+
+                RoundButton {
+                    id: ptButton
+                    radius: 5
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignHCenter
+
+                    Image {
+                        id: ptImage
+                        source: "iconAssets/point-32.png"
+                        anchors {
+                            horizontalCenter: parent.horizontalCenter
+                            verticalCenter: parent.verticalCenter
+                            verticalCenterOffset: -ptText.height/2
+                        }
+                        width: 20
+                        fillMode: Image.PreserveAspectFit
+                    }
+
+                    Text {
+                        id: ptText
+                        text: qsTr("Point")
+                        font.pixelSize: 8
+                        anchors {
+                            top: ptImage.bottom
+                            horizontalCenter: parent.horizontalCenter
+                        }
+                    }
+
+                    onClicked: model.setSketchCreationMode(0);
+                }
+
+                RoundButton {
+                    id: mPtButton
+                    radius: 5
+                    Layout.fillWidth: true
+                    Layout.alignment: Qt.AlignHCenter
+
+                    Image {
+                        id: mptImage1
+                        source: "iconAssets/point-32.png"
+                        anchors {
+                            right: parent.horizontalCenter
+                            rightMargin: -mptImage2.width/5
+                            verticalCenter: parent.verticalCenter
+                            verticalCenterOffset: -mptText.height/2 - 2
+                        }
+                        width: 20
+                        fillMode: Image.PreserveAspectFit
+                    }
+
+                    Image {
+                        id: mptImage2
```

Comment:
why are there 2 images? Only difference I can spot are the anchors

Reply:
I included two images here to illustrate that it's a multipoint geometry. Let me know if there's a better way to do it.
![Screenshot 2021-07-22 114744](https://user-images.githubusercontent.com/48941951/126692728-2ada3840-308d-450b-9cf7-8a0d2cb49808.png)